CHARTER.md

```diff
@@ -309,6 +310,56 @@ o. Any issues that cannot be resolved by the TOB shall be referred to The Linux
    Foundation Executive Director for mediation, and, if necessary, for
    resolution by The Linux Foundation Board of Directors.
 
+p. OCI Working Groups.
+
+   - i. The OCI may establish one or more working groups (each, an “OCI Working
+     Group”) to experiment, validate, and define new OCI Specifications or
+     major revisions to existing OCI Specifications. Each OCI Working Group
+     shall have a defined scope of the OCI Working Group's intended activities
+     and goals. The end result of the OCI Working Group's work product shall be
+     either:
```

Comment:
This part is still a little unclear to me, when a WG is proposing changes to an existing specification that already has existing maintainers.

It seems like the following would be possible with this structure:
* a WG to form to make changes to an existing specification
* The WG decides on, and present these changes to the TOB for approval 
* The TOB ratifies these changes

all without any input from the existing maintainers.

Would it make sense to handle the case where changes are being proposed to existing specification(s) separately from a brand new specification?

Reply:
I tried to answer that above, existing maintainers could be sponsors and existing specs could be dependencies, but not the final result of the working group. I think in the case you mentioned where the working group covers existing specs, the TOB would vote to disband the working group after all the existing specs are updated or the working group cannot make progress because their changes were not accepted by the maintainers.